src/Relude/Extra/Foldable.hs

```diff
@@ -2,7 +2,8 @@
 {-# LANGUAGE BangPatterns #-}
 
 {- |
-Copyright:  (c) 2018-2020 Kowainik
+Copyright:  (c) 2018-2021 Kowainik
+
```

Comment:
I guess this extra line is redundant. Can make the rest of the header non-renderable... 
```suggestion
```